plyer/facades.py

```diff
@@ -82,23 +84,61 @@ def take_video(self, filename, on_complete):
         :type filename: str
         :type on_complete: callable
         '''
+
         self._take_video(filename=filename, on_complete=on_complete)
 
     # private
 
     def _take_picture(self, **kwargs):
         raise NotImplementedError()
-        
+
     def _take_video(self, **kwargs):
         raise NotImplementedError()
 
 
+class Contacts(object):
```

Comment:
1. These two links give you a glance to know how its structured:
http://developer.android.com/images/providers/contacts_structure.png

and scroll to the down to know how look tables:
http://xianminx.github.io/2012/06/24/android-contact-provider-in-deep/

Contacts and other data like emails or groups are in sqlite database. Everything we can get by doing a query.

2. print "I have {} contacts".format(len(Contacts))

I have added that feature